src/arguments.py

```diff
@@ -0,0 +1,17 @@
+import os
+import typing as t
+from argparse import ArgumentParser
+
+
+def create_arguments() -> ArgumentParser:
+    ap = ArgumentParser(description='Module for migrate postgres database')
+    ap.add_argument('-p', '--path', default=os.environ.get('MIGRATOR_MIGRATIONS_PATH'),
+                    required=True, help='Path to migrations')
+    ap.add_argument('-d', '--database', default=os.environ.get('MIGRATOR_DB_URI'),
+                    required=True, help='URI for connect to database')
+    return ap
+
+
+def get_arguments() -> dict[str, t.Any]:
+    ap = create_arguments()
+    return vars(ap.parse_args())
```

Comment:
why u need vars here?

parse_args returns object that have attributes which is you arguments